hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -1938,8 +1937,8 @@ private class Emit(
     x match {
       case NDArrayMap(child, elemName, body) =>
         val childP = child.pType.asInstanceOf[PNDArray]
-        val elemPType = childP.elementType
-        val elemRef = mb.newPField(elemName, elemPType)
+        val elemPType = -childP.elementType
```

Comment:
This I think should not change reauiredeness

Reply:
Removed.